static/src/javascripts/projects/common/modules/navigation/edition-picker.js

```diff
@@ -0,0 +1,67 @@
+define([
+    'common/utils/fastdom-promise',
+    'common/utils/$'
+], function (
+    fastdomPromise,
+    $
+) {
+    var dropdown = $('.js-edition-picker-dropdown');
+
+    function enhanceToButton() {
+        var container = $('.js-edition-picker-container');
+        var checkBox = $('.js-enhance-checkbox');
+        var button = $.create('<button>');
+
+        button.addClass('edition-picker__button js-click');
+        button.attr('id', 'edition-picker');
+        button.attr('aria-controls', 'edition-picker__dropdown');
+        button.attr('aria-expanded', 'true');
+
+        fastdomPromise.write(function() {
+            checkBox.remove();
+            container.prepend(button);
+
+            openMenuOnClick();
+        });
+    }
+
+    function openMenuOnClick() {
+        var button = $('.js-click');
+
+        if (button.length > 0) {
+            button[0].addEventListener('click', function() {
+
+                fastdomPromise.write(function() {
+                    if (button.hasClass('open')) {
+                        button.removeClass('open');
+                        dropdown.attr('hidden');
```

Comment:
Does this set it as expected? I thought a single value in `attr` was a getter only but maybe it does something special with boolean attributes